test/test-jdk.js

```diff
@@ -25,7 +26,11 @@ describe('JDK', () => {
 	});
 
 	it('should error if dir is missing the JVM library', () => {
-		return detect(path.join(__dirname, 'mocks', 'empty'))
+		const mockDir = path.join(__dirname, 'mocks', 'empty');
+		if (!fs.existsSync(mockDir)) {
+			fs.mkdirSync(mockDir);
```

Comment:
I don't understand. The `empty` directory does exist: https://github.com/appcelerator/jdklib/tree/master/test/mocks/empty.

Reply:
Yeah, it does in the repo, it does when we unstash on linux, but when we unstash on OSX/Windows it doesn't, despite using `allowEmpty: true` in the stash (maybe I'm using that wrong, I dunno), see the failures in [this run](https://jenkins.appcelerator.org/blue/organizations/jenkins/cli%2Fjdklib/detail/jenkinsfile/9/pipeline/22) for example

I guess I should add a comment clarifying why I added that.

Reply:
So I think I was using it wrong. I can remove this :)